neo4j/README.md

```diff
@@ -38,26 +41,37 @@ To install the neo4j check on your host:
 
 ### Metrics
 
-See [metadata.csv][6] for a list of metrics provided by this check.
+Neo4j version 4
+Neo4j 4 metrics are collected as documented [here][12]. The most commonly monitored metrics are provided in the out-of-the-box dashboards. 
+
+Neo4j version 5
+Neo4j 5 metrics are collected as documented [here][11]. The most commonly monitored metrics are provided in the out-of-the-box dashboards. 
```

Comment:
```suggestion
- Neo4j 5 metrics are collected as documented [here][11]. The most commonly monitored metrics are provided in the out-of-the-box dashboards. 
```

Reply:
This one might have gotten caught in the update I made to the readme file.